### PR TITLE
SPSH-1086

### PR DESCRIPTION
**Pull request by tikle-cap**
https://ticketsystem.dbildungscloud.de/browse/SPSH-1086

### CODE REVIEW

File: charts/dbildungs-iam-keycloak/dev-realm-spsh.json [hunk 1]
```diff
@@ -454,6 +455,24 @@
...
+        "username": "service-account-spsh-service",
+        "enabled": true,
+        "totp": false,
+        "emailVerified": false,
+        "serviceAccountClientId": "spsh-service",
+        "disableableCredentialTypes": [],
+        "requiredActions": [],
+        "clientRoles": {
+          "realm-management": [
+            "manage-realm"
```
Comment (niklaskoopmann) [inline charts/dbildungs-iam-keycloak/dev-realm-spsh.json:470]: @tikle-cap @kristoff-kiefer We need "manage-clients" here. Sorry for the confusion.

File: charts/dbildungs-iam-keycloak/dev-realm-spsh.json [hunk 2]
```diff
@@ -799,6 +818,109 @@
...
+        "name": "SPSH Service Client",
+        "description": "",
+        "rootUrl": "",
+        "adminUrl": "",
+        "baseUrl": "",
+        "surrogateAuthRequired": false,
+        "enabled": true,
+        "alwaysDisplayInConsole": false,
+        "clientAuthenticatorType": "client-jwt",
+        "secret": "**********",
```
Comment (niklaskoopmann) [inline charts/dbildungs-iam-keycloak/dev-realm-spsh.json:833]: @tikle-cap Please replace asterisks with some plain text client secret (e. g. "spsh-service-client-secret") once we are sure this cannot be abused.

File: charts/dbildungs-iam-keycloak/prod-realm-spsh.json
Comment (niklaskoopmann) [inline charts/dbildungs-iam-keycloak/prod-realm-spsh.json:1]: @tikle-cap Please review changes to this file. This seems to be a lot and partially unnecessary.

**Review by niklaskoopmann — CHANGES_REQUESTED** [review]

File: charts/dbildungs-iam-keycloak/prod-realm-spsh.json
Comment (tikle-cap) [reply charts/dbildungs-iam-keycloak/prod-realm-spsh.json:1]: formatting fixed

File: charts/dbildungs-iam-keycloak/dev-realm-spsh.json (hunk 1 above)
Comment (tikle-cap) [reply charts/dbildungs-iam-keycloak/dev-realm-spsh.json]: replaced

File: charts/dbildungs-iam-keycloak/dev-realm-spsh.json (hunk 2 above)
Comment (tikle-cap) [reply charts/dbildungs-iam-keycloak/dev-realm-spsh.json]: replaced

**Review by niklaskoopmann — APPROVED** [review]

**Pull request merged**